assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -386,10 +387,18 @@ export function getAnalytics4MockResponse(
 	const args = cloneDeep( options );
 
 	const originalSeedValue = faker.seedValue;
-	const argsHash = parseInt(
-		md5( args.url || 'http://example.com' ).substring( 0, 8 ),
-		16
-	);
+	const argsURL = args.url || 'http://example.com';
+
+	let seed = argsURL;
+
+	if (
+		args.dimensionFilters &&
+		isValidDimensionFilters( args.dimensionFilters )
```

Comment:
We should use `invariant()` to throw an error if the passed `args.dimensionFilters` aren't valid, otherwise we are less likely to catch mistakes.

This should include test coverage as we do test the validation of the other options.